src/v2/Components/ArtworkFilter/ArtworkFilters/index.tsx

```diff
@@ -14,30 +14,36 @@ import { ArtworkLocationFilter } from "./ArtworkLocationFilter"
 import { ArtistNationalityFilter } from "./ArtistNationalityFilter"
 import { MaterialsFilter } from "./MaterialsFilter"
 import { PartnersFilter } from "./PartnersFilter"
+import { Box } from "@artsy/palette"
+import { ScrollRefContext } from "./useScrollContext"
 
 export const ArtworkFilters: React.FC = () => {
   const showNewFilters = getENV("ENABLE_NEW_ARTWORK_FILTERS")
 
+  const scrollRef = useRef(null)
+
   return (
-    <>
-      <MediumFilter expanded />
-      {showNewFilters && <MaterialsFilter expanded />}
-      <PriceRangeFilter />
-      <AttributionClassFilter expanded />
-      {showNewFilters ? <SizeFilter2 /> : <SizeFilter />}
-      <WaysToBuyFilter />
-      {showNewFilters && <ArtworkLocationFilter expanded />}
-      {showNewFilters && <ArtistNationalityFilter expanded />}
-      <TimePeriodFilter />
-      <ColorFilter />
-      {showNewFilters ? (
-        <PartnersFilter />
-      ) : (
-        <>
-          <GalleryFilter />
-          <InstitutionFilter />
-        </>
-      )}
-    </>
+    <Box ref={scrollRef} overflowY="scroll" height="100%">
```

Comment:
This `Box` is not supposed to be on desktop, so far I'm just trying to make it work correctly on mobile